lib/request.js

```diff
@@ -257,15 +257,14 @@ module.exports = {
    */
 
   get fresh() {
-    const method = this.method;
-    const s = this.ctx.status;
+    const { method, ctx: { status: s, response }, header } = this;
 
     // GET or HEAD for weak freshness validation only
-    if ('GET' != method && 'HEAD' != method) return false;
+    if (!['GET', 'HEAD'].includes(method)) return false;
```

Comment:
In original code yoy have 3 logical operators and must be sure in prorities of it. In refactored code yoy have one logical function. I can understand if you will call it 'I don't want to learn new Array methods', but 'drop of readability'?